### DISCUSSION TITLE
"Specify transparent color" setting disappears

### DISCUSSION THREAD

**Discussion by hiroto-nagaoka** [PHPOffice/PhpSpreadsheet#4117] [Q&A] [answered]
Process of loading Excel, converting arbitrary values, and outputting them
I had set "Specify transparent color" in the Excel I was importing, but the setting was canceled after outputting.

**Comment by oleibman** [comment]
This is not (yet) supported. Please open a new issue (feature request) if you would like us to investigate supporting it.

**Reply by hiroto-nagaoka** [reply]
Sorry for being late.
Thanks for the answer!

**Reply by oleibman** [reply, marked as answer]
@hiroto-nagaoka PR #4142 addresses your question, and is now merged.
